\documentclass[a4paper,11pt,oneside]{article}
\usepackage{a4wide}                     % Iets meer tekst op een bladzijde
\usepackage[english]{babel}               % Voor nederlandstalige hyphenatie (woordsplitsing)
\usepackage{amsmath}                    % Uitgebreide wiskundige mogelijkheden
\usepackage{amssymb}                    % Voor speciale symbolen zoals de verzameling Z, R...
\usepackage{url}                        % Om url's te verwerken
\usepackage[round]{natbib}
\usepackage{graphicx}                   % Om figuren te kunnen verwerken
\usepackage[small,bf,hang]{caption}    % Om de captions wat te verbeteren
\usepackage{xspace}                     % Magische spaties na een commando
\usepackage[latin1]{inputenc}           % Om niet ascii karakters rechtstreeks te kunnen typen
\usepackage{float}                      % Om nieuwe float environments aan te maken. Ook optie H!
\usepackage{flafter}                    % Opdat floats niet zouden voorsteken
\usepackage{listings}                   % Voor het weergeven van letterlijke text en codelistings
\usepackage{marvosym}                   % Om het euro symbool te krijgen
\usepackage{textcomp}                   % Voor onder andere graden celsius
\usepackage{hyperref}
\usepackage{fancyhdr}                   % Voor fancy headers en footers.
\usepackage{graphics}			% Om figuren te verwerken.
%\usepackage{biblatex}
\bibliographystyle{biblioeng2}
\newcommand{\npar}{\par \vspace{2.3ex plus 0.3ex minus 0.3ex} \noindent}	% Om witruimte te krijgen tussen paragrafen
\hyphenation{stu-den-ten-ver-te-gen-woor-di-ger op-lei-dings-com-mis-sies com-pu-ter-com-mis-sies fa-cul-teits-raad 
In-ge-nieurs-We-ten-schap-pen}
\graphicspath{{figuren/}}               % De plaars waar latex zijn figuren gaat halen.

% Nieuw commando om figuren in te voegen. Gebruik:
% \mijnfiguur[H]{width=5cm}{bestandsnaam}{Het bijschrift bij deze figuur}
\newcommand{\mijnfiguur}[4][ht]{            % Het eerste argument is standaar `ht'.
    \begin{figure}[#1]                      % Beginnen van de figure omgeving
        \begin{center}                      % Beginnen van de center omgeving
            \includegraphics[#2]{#3}        % Het eigenlijk invoegen van de figuur (2: opties, 3: bestandsnaam)
            \caption{#4\label{#3}}          % Het bijschrift (argument 4) en het label (argument 3)
        \end{center}
    \end{figure}
    }
%opening
\title{Curriculum Vitae}
\author{Frederiek - Maarten Kerckhof}

\begin{document}

\maketitle

\section*{Personalia}
\begin{itemize}
  \item Last name: Kerckhof
  \item First name: Frederiek - Maarten
  \item Current adress: Brankardierstraat 44, 9000 Gent
  \item Mobile phone: +32477/98.13.12
  \item E-mail: frederiekmaarten.kerckhof@gmail.com
  \item Date of birth: 14 November 1988
\end{itemize}
\section*{Research interests}
I would very much like to conduct a research-oriented task in the environmental biotechnology domain. Due to my doctoral research and training as a bioscience engineer option cellular and genetic biotechnology I consider myself qualified for a function in research and development. My social and organisatorial skills make me a good collaborator in a research team in which, if necessary, I can take a coordinating task. I have guided or assisted in the guidance of 4 master thesis students and 1 intern, showing that I can delegate my tasks and organize and coordinate them in a research environment.  My main interests in environmental biotechnology lie in unravelling microbial interactions that underly the major reactions in nutrient cycling. In my belief the deeper understanding of these complex multicellular behaviour of unicellular organisms will reveal interesting perspectives in engineered ecosystems. Synthetic ecology is a key method in unravelling these interactions. The understanding of the operation of a microbial metabolic network  as opposed to the black-box catch-all term 'biomass' that is performing the reactions in e.g. methane oxidation could allow for steering of methane oxidation rates and valuable by-products formed by methane oxidation partners. My bio-informatics and statistics experience in a great variety of projects in microbial ecological research show that I can be of added value in this research either from a theoretical and applied point of view.
\section*{Education}
\begin{itemize}
\item \textbf{2011-present: Doctoral thesis on sustainable methanotrophy}
    \begin{itemize}
      \item Academic papers as first author (see bibliography below, \cite{kerckhof2014optimized})
      \item Many co-authorships as bio-informatics or statistical consultant  (see bibliography below)
      \item Guided master thesis and internship students
      \item Teaching assistant in practical exercises molecular microbial techniques and microbial ecological processes.
      \item organizing internal courses on statistics, bio-informatics and version control
    \end{itemize}
\item \textbf{2011-present: Ghent university doctoral schools}
    \begin{itemize}
      \item Advanced academic English: conference skills
      \item IVPV specialist course: Advanced statistical methods - nonparametric methods
      \item IVPV specialist course: Advanced statistical methods - multivariate methods
      \item FLAMES specialist course: Advanced R - Programming in R and beyond
      \item Specialist course: UGent High Performance computing (Linux shell scripting, Python, HPC usage)
      \item Specialist course: Introduction to MG-RAST
    \end{itemize}
\item \textbf{2006-2011: Msc. Bioscience engineering in cellular and genetic biotechnology, major computational biology at Ghent University}. 
	\begin{itemize}
		\item Bachelorpaper: 'Competition and diversity: apparent opposites?'. About (mathematic modelling of) ecological competition on both macro- and micro-ecological levels and possible applications (preemptive colonisation, pre- and pro-biotics).
		\item Combined project statistics for genome analysis and bio-informatics: analysis of 454-pyrosequencing metagenomics data
		\item Masters thesis: 'The impact of the physical state of electron donors and acceptors on microbial fysiology and morphology'. Fundamental research concerning microbial electron metabolism in bio-electrochemical systems (microbial fuel cells).
	\end{itemize}
\item 2000-2006: Latin - Mathematics, Onze-Lieve-Vrouwecollege Assebroek
\end{itemize}

\section*{Work experience}
\begin{itemize}
\item Volunteer work
	\begin{itemize}
	\item 2006-2010: Scout leader.
	\item 2007-2009: Materials manager in the Scouts.
  \item 2010-2013: Group leader in a team of 3 for Scouts Don Bosco, managin a staff of 30-40 leaders responsible for 200-250 children with weekly activities and yearly camp
  \item 2012-2014: District commisary for Scouts District 't Brugse Vrije, pedagogic coordinator of 12 scout groups with 350 leaders and 2000+ children within Bruges
	\end{itemize}
\item Extracurricular activities
	\begin{itemize}
	\item 2004-2006: Editorial and technical staff at OINC-TV (secondary school). 
	\item 2004-2006: Chosen representative in the student council, work group coordinator.
	\item 2009-2011: Chosen year-representative cellular and genetic biotechnology (arranging exam schedule, have a seat in the faculty's student council).
	\end{itemize}
\end{itemize}

\section*{Teaching experience}
\begin{itemize}
  \item 2011-2013 Practical exercises molecular microbial techniques (Msc Bioscience engineering)
  \item 2013-2015 Practical exercises microbial ecological processes (Bsc Bioscience engineering)
\end{itemize}

\section*{Skills}
\subsection*{Informatics}
As a major in computational biology I sure know my way around computers, below are listed some of the specific software applications that I am familiar with.
\begin{itemize}
	\item R, bioconductor and many R packages for statistical analysis.
	\item The Mathworks Matlab and simulink for modelling and advanced mathematics.
	\item Perl/Bioperl \& Python/Biopython for scripting.
	\item ImageJ and comstat for bio-imaging.
  \item Advanced in mothur and knowledge of Qiime for NGS data analysis
  \item RAxML for phylogenetic tree building
  \item MG-RAST for metagenomics
	\item Basic knowledge of Bionumerics, DNAStar lasergenes, PROKKA, CLC workbench, Wolfram Mathematica,  Ruby, Java, HTML, php and Visual Basic.
\end{itemize}
\subsection*{Lab skills}
During my master's thesis at LABMET I had the opportunity to be introduced to many methods in microbiological research, which I further developped during my doctoral thesis research.
\begin{itemize}
	\item Common chemical analsytical techniques (VSS/TSS, Kjeldahl-N, COD, CDW, GC-VFA, HPLC, $\ldots$).
	\item Common molecular techniques (PCR, DGGE, qPCR, Illumina MiSeq amplicon data analysis).
	\item Use of a flowcytometer and confocal microscopy.
  \item Programming and use of Tecan Freedom EVO liquid handling system.
	\item Biosafety training for a BSL-3 laboratory.
\end{itemize}
\section*{Online Presence}
\begin{itemize}
  \item \href{https://www.linkedin.com/pub/frederiek-maarten-kerckhof/26/b47/668}{LinkedIn}
  \item \href{https://www.researchgate.net/profile/Frederiek-Maarten_Kerckhof}{ResearchGate}
  \item \href{http://www.researcherid.com/ProfileView.action?SID=V2bGbhtEe1TlsfIEXBz&returnCode=ROUTER.Success&queryString=KG0UuZjN5WlUD2sX8KoC12Tw17vPT2A6ocQ5tgzRDDI\%253D\&SrcApp=CR\&Init=Yes}{ResearcherID}
  \item \href{http://orcid.org/0000-0002-4472-6810}{ORCID}
  \item Contributor to stack Exchange fora (StackOverflow, CrossValidated, Ask Ubuntu, TeX)
\end{itemize}

\renewcommand{\refname}{Academic publications and conference proceedings} %(voor article)
\renewcommand{\bibname}{Academic publications and conference proceedings} %(voor book en report)
\nocite{*}

\bibliography{citations}
%\addbibresource{citations.bib}
%\printbibliography
\end{document}